common/cgroup/v1/cgroup.go

```diff
@@ -0,0 +1,419 @@
+// Copyright 2016 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//+build linux
+
+package v1
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"syscall"
+
+	"github.com/hashicorp/errwrap"
+)
+
+var (
+	cgroupControllerRWFiles = map[string][]string{
+		"memory":  {"memory.limit_in_bytes"},
+		"cpu":     {"cpu.cfs_quota_us", "cpu.shares"},
+		"devices": {"devices.allow", "devices.deny"},
+	}
+)
+
+func mountFsRO(mountPoint string) error {
```

Comment:
In see `MS_STRICTATIME` set in https://github.com/coreos/rkt/pull/3277/files#diff-255755d0e4a8ba8def22e94ff6f0c1b6R276 which is not listed here.